### DISCUSSION TITLE
How to bind ctrl+shift+backslash

### DISCUSSION THREAD

**Discussion by IcyAlmond** [kovidgoyal/kitty#5497] [Q&A] [answered]
I'm trying to do this:
`map ctrl+shift+backslash --location=vsplit`
I've also tried this
`map ctrl+| --location=vsplit`
`map ctrl+shift+\ --location=vsplit`
`map ctrl+shift+0x2b --location=vsplit`
But whenever I press ctrl+shift and backslash at the same time, it just types the `|` character as normal

**Comment by IcyAlmond** [comment, marked as answer]
i forgot the `launch` part. as in:
`map ctrl+shift+backslash launch --location=vsplit`

**Comment by kovidgoyal** [comment]
map ctrl+shift+\ launch --location=vsplit

On Sat, Sep 10, 2022 at 10:09:14PM -0700, Latrolage wrote:
> I'm trying to do this:
> `map ctrl+shift+backslash --location=vsplit`
> I've also tried this
> `map ctrl+| --location=vsplit`
> and
> `map ctrl+shift+\ --location=vsplit`
> But whenever I press ctrl+shift and backslash at the same time, it just types the `|` character as normal
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/5497
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________
